### PR TITLE
Documentation cleanup 2 + small consumer's requests

### PR DESCRIPTION
**Pull request by ala-n**
- Move/restructure contributors documentation
- Documentation rework actualization for toggleable, fix `togglable` variant
- Documentation for triggers
- Base trigger moved to trigger module
- Feature: add the ability to control default event preventing for all triggers
- Feature: add the ability to choose a target for ESLMedia ready class

### CODE REVIEW

File: docs/contribute/commit.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Commit Convention
 
+Project is using `@commitlint/config-conventional` commit message rules. Please check it out in case you want to
+contribute to ESL. Use this short note to renew commit message rules.
+
+In case you are using JetBrains IDE (IDEA or WebStorm) you can use this
+[plugin](https://plugins.jetbrains.com/plugin/13389-conventional-commit) to simplify commit validation.
```
Comment (dshovchko) [inline docs/contribute/commit.md:7]: Please, add https://marketplace.visualstudio.com/items?itemName=vivaxy.vscode-conventional-commits for Visual Studio Code

File: src/modules/esl-toggleable/README.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 # [ESL](../../../README.md) Toggleable
 
-Version: *1.0.0-beta*.
-
-***Important Notice: the component is under beta version, it was tested and now ready to use but be aware of its potential critical API changes.***
+Version: *1.0.0*.
```
Comment (dshovchko) [inline src/modules/esl-toggleable/README.md:3]: The point at the end of sentences. Either delete it here or return it to the rest.

File: src/modules/esl-trigger/README.md [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
-Version: *1.0.0-beta*.  
-***Important Notice: the component is under beta version, it tested and ready to use but be aware of its potential critical API changes.***
+Version: *1.0.0*.
 
 Authors: *Alexey Stsefanovich (ala'n)*, *Julia Murashko*.
 
 ESLTrigger - custom element, that allows to trigger ESLToggleable instances state changes.
 
-Description TBD.
+ESLBaseTrigger - base class for a custom element, that allows to trigger ESLToggleable instances state changes.
```
Comment (dshovchko) [inline src/modules/esl-trigger/README.md:9]: ESLBaseTrigger - base class for a custom element, that allows triggering ESLToggleable instances state changes.

File: src/modules/esl-trigger/README.md [hunk 4]
```diff
@@ -1,10 +1,37 @@
...
+
+- `a11y-target` - selector of inner target element to place aria attributes. Uses trigger itself if blank
+
+- `active-class` - CSS classes to set on active state
+
+- `active-class-target` - target element TraversingQuery selector to set `active-class`
+
+### ESLToggleable Attributes / Properties
+
+- `target` - target Toggleable TraversingQuery selector. `next` by default
```
Comment (dshovchko) [inline src/modules/esl-trigger/README.md:25]: `next` by default
Maybe you should put it in brackets?

**Review by dshovchko — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Pull request merged**